src/TestHostExtension/stdafx.h

```diff
@@ -8,6 +8,9 @@
 
 #pragma once
 
+#define _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING
+#include <experimental/filesystem>
```

Comment:
>experimental [](start = 10, length = 12)

Was filesystem moved to non-experimental? Isn't that the one we would want to be using?

Reply:
The <filesystem> in the 141 toolset was actually the experimental one. I'll create an issue

Reply:
https://docs.microsoft.com/en-us/cpp/standard-library/filesystem?view=vs-2019

Reply:
#145